src/components/organisms/cards/cards.html

```diff
@@ -0,0 +1,68 @@
+<div class="cards">
+  <div class="card-body container">
+    <div class="row row-cols-1 row-cols-md-3">
+      <div class="col">
+        <div class="card">
+          <p class="title">CONCEPT</p>
+          <figure class="img-wrapper">
+            <img src="../../../img/card-1.jpg" class="card-img" />
+          </figure>
+          <div class="card-triangle"></div>
+          <div class="card-body">
+            <p class="card-text">
+              「本屋」でも「自習室」でもない、新しい空間。それがこのカフェ＆コワーキングです。
+              <br />お好きなドリンクを飲みながらゆったりと気兼ねなく勉強や仕事に取り組んでいただけます。
+            </p>
```

Comment:
```suggestion
            <p class="card-text card-text-1">
              「本屋」でも「自習室」でもない、新しい空間。それがこのカフェ＆コワーキングです。
              <br />お好きなドリンクを飲みながらゆったりと気兼ねなく勉強や仕事に取り組んでいただけます。
            </p>
```
card-textクラスにpaddingを追加すると、3つのカード全てにpaddingが追加されてしまい、高さの調節ができません。そこで、クラス名を増やしましょう。他のカードも同様にcard-text-2などとして、そこにそれぞれpaddingを当てれば良いと思います。

Reply:
ちなみにbreakpointは以下のように二つの間に適用することもできます。
```
          @media (min-width: $breakpoint-lg) and (max-width: $breakpoint-xl) 
```